curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/find-characters-with-lazy-matching.english.md

```diff
@@ -26,7 +26,10 @@ Fix the regex <code>/&lt;.*&gt;/</code> to return the HTML tag <code>&lt;h1&gt;<
 tests:
   - text: The <code>result</code> variable should be an array with <code>&lt;h1&gt;</code> in it
     testString: assert(result[0] == '<h1>');
-
+  - text: <code>myRegex</code> should use lazy matching
+    testString: assert(myRegex.test(myRegex.source))
```

Comment:
This test passes with the default code and the code in the screenshot.
I think something like this is probably enough to make sure lazy loading was used...
```suggestion
    testString: assert(/\?/g.test(myRegex))
```

Reply:
@moT01 Thanks for that. Made changes